### PR TITLE
gh-119535: Support 𝜋thon in Python 3.14 venvs

### PR DESCRIPTION
**Pull request by foreignmeloman**
* Issue: gh-119535

### CODE REVIEW

**Comment by skirpichev** [comment]
I would guess, it's already too late;)  And sounds wrongly: "Pi", not "Py".

**Comment by hugovk** [comment]
It's not too late.

In English, the py of [python](https://en.wiktionary.org/wiki/python) and [pi](https://en.wiktionary.org/wiki/pi) are pronounced the same: `/paɪ/` in IPA.

**Comment by thatbirdguythatuknownot** [comment]
Also doesn't reveal the easter egg to terminal autocomplete (as far as i know autocomplete goes)

**Comment by foreignmeloman** [comment]
> Also doesn't reveal the easter egg to terminal autocomplete (as far as i know autocomplete goes)

Was it intended to show during the tab completion?

File: Lib/venv/__init__.py [hunk 1]
```diff
@@ -306,7 +306,7 @@ def setup_python(self, context):
 
             suffixes = ['python', 'python3', f'python3.{sys.version_info[1]}']
             if sys.version_info[:2] == (3, 14):
-                suffixes.append('python𝜋')
+                suffixes.append('𝜋thon')
```
Comment (Eclips4) [inline Lib/venv/__init__.py:309]: Why can't we have both of them? :smile:

Comment (foreignmeloman) [reply Lib/venv/__init__.py:309]: I think we should decide on one option. An easter egg should be unique, otherwise it'll loose its punch.

Comment (picnixz) [reply Lib/venv/__init__.py:309]: (I suggest creating the concept of an easter omlet)

**Review by hauntsaninja — COMMENTED** [review]
Hugo asked me if I have an opinion on this. I don't have a strong opinion, but for whatever it's worth I chose `python𝜋` in the original PR because:
- I think it's much easier to use, since your tab completion is narrowed to like four options. Especially so after the switch from U+03c0 to U+1d70b (I can type U+03c0 on my keyboard pretty easily but not U+1d70b)
- Subjective, but to me some of the joke comes from the substitution of 3.14 in the "python3.14" file that venv creates. It's clearer that it's specific to this specific version of Python

**Comment by hugovk** [comment]
Okay, let's go for this one :)

Remember, everyone: shhh!

File: Lib/venv/__init__.py (hunk 1 above)
Comment (doekman) [reply Lib/venv/__init__.py:309]: Both would be better. `𝜋thon` is the best of course, but `python𝜋` is discoverable via tab-completion.

Comment (i-am-called-glitchy) [reply Lib/venv/__init__.py:309]: An easter egg *should* be also hidden

> Both would be better. `𝜋thon` is the best of course, but `python𝜋` is discoverable via tab-completion.

**Comment by secolive** [comment]
I'm having oooh such a blast with this easter egg :
[sender] cannot convert filename: libexec/bin/𝜋thon (Illegal byte sequence)

**Pull request merged**